### ISSUE TITLE
j - jump to a specific block number / block hash

### ISSUE THREAD

**Issue by lithp** [lithp/turtlescan#1]
Currently the list of blocks only shows you the most recent block, and doesn't allow you to scroll past the end. It would be nice to have a way of jumping to some desired part of the blockchain.

"j" should open some kind of popup, prompting you to enter an absolute or relative offset, and then scroll the viewport to the requested block

This likely depends on #2
